runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/Table/components/TableRowExpander.tsx

```diff
@@ -12,7 +12,7 @@ import { SUBROW_INDENT_MULTIPLIER } from '../Table.utils';
 const TableRowExpander: VFC<TableRowExpanderProps> = ({ row, handleClick }) => {
     const { translate } = useTranslations();
     return (
-        <RowCustomExpandedSpan isExpanded={row.isExpanded} depthIndent={row.depth * SUBROW_INDENT_MULTIPLIER} onClick={() => handleClick(row)}>
+        <RowCustomExpandedSpan isExpanded={row.isExpanded} depthIndent={row.depth * SUBROW_INDENT_MULTIPLIER} onClick={() => handleClick?.(row)}>
```

Comment:
I think it could be nice to check if handleClick was provided before we want to execute onClick